### PR TITLE
Remove unused code for nonexistent json configuration.

### PR DESCRIPTION
**Pull request by jplevyak**
Signed-off-by: John Plevyak <jplevyak@gmail.com>

### CODE REVIEW

File: source/extensions/access_loggers/wasm/config.cc [hunk 1]
```diff
@@ -26,6 +26,9 @@ WasmAccessLogFactory::createAccessLogInstance(const Protobuf::Message& proto_con
   const auto& config = MessageUtil::downcastAndValidate<
       const envoy::extensions::access_loggers::wasm::v3::WasmAccessLog&>(
       proto_config, context.messageValidationVisitor());
+
+  AccessLog::FormatterPtr formatter;
```
Comment (PiotrSikora) [inline source/extensions/access_loggers/wasm/config.cc:30]: Is this related change?

Comment (jplevyak) [reply source/extensions/access_loggers/wasm/config.cc]: Nice catch nope.  Removed.
